apps/web/src/client/components/settings/providers/useOAuthSignIn.ts

```
@@ -0,0 +1,149 @@
+          return;
+        }
+        logger.error('Error polling OpenAI OAuth status:', err);
+        setError(err instanceof Error ? err.message : t('status.signInFailed'));
+        setSigningIn(false);
+      });
+    } catch (err) {
+      if (abortController.signal.aborted || attemptId !== signInAttemptRef.current) return;
+      setError(err instanceof Error ? err.message : t('status.signInFailed'));
+    } finally {
+      if (!pollStarted && (abortController.signal.aborted || attemptId !== signInAttemptRef.current)) {
+        return;
```

Biome (2.4.9)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)